manifests/r/RazerInc/Synapse/1.17.0.597/RazerInc.Synapse.yaml

```
@@ -0,0 +1,8 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: RazerInc.Synapse

```

```suggestion
PackageIdentifier: RazerInc.RazerInstaller.Beta
```
Please update file name and path to match this change.